modules/smaatoBidAdapter.md

```diff
@@ -0,0 +1,64 @@
+# Overview
+
+```
+Module Name: Smaato Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: prebid@smaato.com
+```
+
+# Description
+
+The Smaato adapter requires setup and approval from the Smaato team, even for existing Smaato publishers. Please reach out to your account team or prebid@smaato.com for more information.
+
+# Test Parameters
+
+For banner adunits:
+
+```
+var adUnits = [{
+    "code": "banner-unit",
+    "mediaTypes": {
+        "banner": {
+            "sizes": [320, 50]
+        }
+    },
+    "bids": [{
+        "bidder": "smaato",
+        "params": {
+            "publisherId": "1100012345",
+            "adspaceId": "11002234"
```

Comment:
Please provide values in the README that can be used in a test page and will return data.

Using this `adspaceId`, I received a 400 Bad Request response from your server with the following headers:
```
X-SMT-MESSAGE: Unknown or blocked or deleted adspace: 11002234
X-SMT-SessionId: 22f18133-8d3a-4568-a16d-ee059bb92503
```

Reply:
I changed the README and added a valid publisherId and adspaceId. It's a bit tricky to test, since there is no publicly available test environment, which will always return a bid. So you will need to send real bid requests and hope for bid responses. Using publisherId=1100042525 and adspaceId=130563103 you sometimes may get a banner ad. You need to make sure you're simulating a mobile web request, by e.g. using Chrome's device toolbar.